weave-js/src/components/Panel2/ChildPanelExportReport/ChildPanelExportReport.tsx

```diff
@@ -156,14 +163,21 @@ export const ChildPanelExportReport = ({
   };
 
   const onDiscardDraft = async () => {
-    if (!hasActiveDraft || !slateNode) {
+    if (!hasActiveDraft || !hasRequiredData) {
       return;
     }
     try {
       await deleteReportDraft({
         variables: {id: activeReportDraft.id},
       });
-      await submit(newDraftVariables(publishedReport, slateNode));
+      await submit(
+        newDraftVariables(
+          selectedEntity?.name,
```

Comment:
Could these just be `.` instead of `?.` - seems like the `hasRequiredData` check should handle it?